packages/client/modules/teamDashboard/components/ProviderRow/JiraServerProviderRow.tsx

```diff
@@ -155,9 +163,32 @@ const JiraServerProviderRow = (props: Props) => {
           </ListAndMenu>
         ) : (
           <ProviderActions>
-            <StyledButton key='linkAccount' onClick={openOAuth} palette='warm' waiting={submitting}>
-              {isDesktop ? 'Connect' : <Icon>add</Icon>}
-            </StyledButton>
+            {provider ? (
+              <StyledButton
+                key='linkAccount'
+                onClick={openOAuth}
+                palette='warm'
+                waiting={submitting}
+              >
+                {isDesktop ? 'Connect' : <Icon>add</Icon>}
+              </StyledButton>
+            ) : (
+              <Form
+                method='get'
+                target='_blank'
+                action={ExternalLinks.CONTACT}
+                onSubmit={() => {
+                  SendClientSegmentEventMutation(atmosphere, 'Clicked Jira Server Request Button', {
+                    viewerId: id
```

Comment:
-1 I missed this before, but this is the team member id, not the viewer user id.